DEV(RI): add documentation for Redis Copilot (#315)
redis/docs: content/develop/connect/insight/_index.md

<img src="images/Databases.png">

### Redis Copilot

Redis Copilot is an AI-powered developer assistant that helps you learn about Redis, explore your Redis data, and build search queries in a conversational manner. It is available in Redis Insight as well as within the Redis public documentation.

Currently, Redis Copilot provides two primary features: a general chatbot and a context-aware data chatbot.

> **General chatbot**: the knowledge-based chatbot serves as an interactive and dynamic documentation interface where developers can ask specific questions about the Redis commands, concepts, and products, and get responses on the fly, simplifying the learning process. The general chatbot is also available in our public docs.

Reviewer: Maybe this sentence is a bit long and complicated as it stands?
```suggestion
**General chatbot**: the knowledge-based chatbot serves as an interactive and dynamic documentation interface to simplify the learning process. You can ask specific questions about Redis commands, concepts, and products, and get responses on the fly. The general chatbot is also available in our public docs.
```

> **My data chatbot**: the context-aware chatbot available in Redis Insight enables users to construct search queries using everyday language rather than requiring specific programming syntax. This feature allows for intuitive query building and data exploring making it easier for developers to interact with their data without extensive technical knowledge.

Reviewer: ```suggestion
**My data chatbot**: the context-aware chatbot available in Redis Insight lets you construct search queries using everyday language rather than requiring specific programming syntax. This feature lets you query and explore data easily and interactively without extensive technical knowledge.
```
